### PR TITLE
fix `Test freezes with "Maximum call stack size exceeded" error - Testcafe version 1.8.3 and up` (close #2405)

### LINKED ISSUE #2405: Test freezes with "Maximum call stack size exceeded" error - Testcafe version 1.8.3 and up
### What is your Scenario?
My test does the following: goes to a product page, opens a Write Review modal, and writes a review. But, when it goes to do anything on the modal (select a star, type the review, etc), it freezes and throws a "Maximum call stack size exceeded" error in the console over and over until I manually quit the test.
I'm unable to upgrade testcafe past 1.8.2 because of this bug. 
 
### What is the Current behavior?
Write Review modal opens, and app freezes when trying to click a rating
 
### What is the Expected behavior?
Modal should not freeze, and should select a rating correctly
 
### What is your public web site URL?

<details>
<summary>Your complete app code (or attach your test files):</summary>

 
```js
fixture('Reviews');
test('Add Review', async (t) => {

    await t.navigateTo('https://www.anthropologie.com/shop/liv-duster-kimono');
    await t
        .click('.c-pwa-reviews-snippet__reviews-link')
        .click('.c-pwa-reviews-snippet__reviews-link'); // might not need this depending on network speed

    await t
        .click('[for="review-rating-1"]')
        .typeText('#review-title', 'test')
        .typeText('[data-qa-review-message]', 'test test test test test test');
});


```
</details>

<details>
<summary>Screenshots:</summary>
Screen Shot 2020-08-13 at 11 36 58 AM

</details>

### Steps to Reproduce:
* Run the above test script
* When the modal opens, you'll see it get frozen when it tries to click on a star, and the console consistently throws the error attached to this ticket
 
### Your Environment details:

* node.js version:                    10.13.0
* browser name and version: Chrome 83
* platform and version:          macOS 10.14
* other:                                   Happens with testcafe v1.8.3 and above (testcafe v1.8.2 does not have this issue)

### PR DESCRIPTION
**Pull request by LavrovArtem**

### CODE REVIEW

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Review by Farfurix — APPROVED** [review]

File: src/client/sandbox/upload/index.ts [hunk 1]
```diff
@@ -93,13 +93,9 @@ export default class UploadSandbox extends SandboxBase {
...
 
-    setUploadElementValue (input: HTMLInputElement, value: string) {
-        if (value === '') {
-            if (UploadSandbox._getCurrentInfoManager(input).clearUploadInfo(input) && isIE && browserVersion > 10)
-                this._riseChangeEvent(input);
-        }
-
-        return value;
+    setUploadElementValue (input: HTMLInputElement, value: string): void {
+        if (value === '' && UploadSandbox._getCurrentInfoManager(input).clearUploadInfo(input) && isIE && browserVersion > 10)
```
Comment (miherlosev) [inline src/client/sandbox/upload/index.ts:97]: `if (value === '' && UploadSandbox._getCurrentInfoManager(input).clearUploadInfo(input) && isIE11)`

File: test/client/fixtures/sandbox/node/attributes-test.js [hunk 2]
```diff
@@ -7,9 +7,10 @@ var destLocation     = hammerhead.get('./utils/destination-location');
...
-var nativeMethods = hammerhead.nativeMethods;
-var browserUtils  = hammerhead.utils.browser;
-var unloadSandbox = hammerhead.sandbox.event.unload;
+var elementEditingWatcher = hammerhead.sandbox.event.elementEditingWatcher;
+var nativeMethods         = hammerhead.nativeMethods;
+var browserUtils          = hammerhead.utils.browser;
+var unloadSandbox         = hammerhead.sandbox.event.unload;
 
 // NOTE: IE11 has a strange bug that does not allow this test to pass
 if (!browserUtils.isIE || browserUtils.version !== 11) {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/node/attributes-test.js:16]: `if (!browserUtils.isIE11) {`

File: test/client/fixtures/sandbox/node/attributes-test.js [hunk 3]
```diff
@@ -1234,4 +1231,24 @@ if (nativeMethods.linkAsSetter) {
     });
 }
 
+if (browserUtils.isChrome) {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/node/attributes-test.js:1234]: Add note why we run this test only for Chrome.

**Review by miherlosev — CHANGES_REQUESTED** [review]

**Review by miherlosev — APPROVED** [review]

**Comment by LavrovArtem** [comment]
@testcafe-build-bot retest

**Pull request merged**